evadb/third_party/databases/slack/__init__.py

```diff
@@ -0,0 +1,15 @@
+# coding=utf-8
+# Copyright 2018-2023 EvaDB
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Database integrations/slack"""
```

Comment:
@kaushikravichandran @jiashenC Do you think we should move this to a new integration folder and call it applications?

Reply:
Yeah, that would be cleaner

Reply:
Moved it to a separate `applications` dir

Reply:
Maybe https://github.com/georgia-tech-db/evadb/tree/staging/apps is a better place. Though the current slack bot does not use any functionality provided by EvaDB, the vision that a bot for answering questions from PDFs seems to be an application built on EvaDB.

Reply:
Moved